File: src/main/java/org/jdc/template/model/database/other/individuallist/IndividualList.java
/*
 * IndividualList.java
 *
 * Created: 03/10/2014 11:15:44
 */



package org.jdc.template.model.database.other.individuallist;

import android.database.Cursor;

import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues;


public class IndividualList extends IndividualListBaseRecord {


    public IndividualList(Cursor cursor) {
        setContent(cursor);
    }

    public IndividualList(DBToolsContentValues values) {
        setContent(values);
    }

    public IndividualList() {
    }


}

File: src/main/kotlin/org/jdc/template/model/database/other/individuallist/IndividualList.kt
/*
 * IndividualList.kt
 *
 * Created: 02/14/2016 07:02:10
 */



package org.jdc.template.model.database.other.individuallist

import android.database.Cursor
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues


class IndividualList : IndividualListBaseRecord {


    constructor() {
    }

    constructor(record: IndividualList) : super(record) {
    }

    constructor(cursor: Cursor) {
        setContent(cursor)
    }

    constructor(values: DBToolsContentValues<*>) {
        setContent(values)
    }


}